content/en/docs/concepts/workloads/management.md

```diff
@@ -32,9 +32,9 @@ service/my-nginx-svc created
 deployment.apps/my-nginx created
 ```
 
-The resources will be created in the order they appear in the file. Therefore, it's best to
-specify the service first, since that will ensure the scheduler can spread the pods associated
-with the service as they are created by the controller(s), such as Deployment.
+The resources will be created in the order they appear in the manifest. Therefore, it's best to
+specify the Service first, since that will ensure the scheduler can spread the pods associated
+with the Service as they are created by the controller(s), such as Deployment.
```

Comment:
I'm not convinced this is a great example; a better one might be a ConfigMap before the Deployment. However, I propose we save that for a follow-up PR.